proto/warehouse/warehouse.proto

```diff
@@ -73,6 +73,7 @@ message WHUploadResponse {
   int32 duration = 14;
   repeated WHTable tables = 15;
   bool isArchivedUpload = 16;
+  int32 status_code = 17;
```

Comment:
I am a bit concern with adding HTTP semantics into GRPC, is it because we are using an HTTP API at some part of the chain ?

Reply:
Let me check how should we remove this.